x-pack/solutions/security/plugins/security_solution/public/siem_migrations/dashboards/components/dashboard_table/utils/filters.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FilterOptionsBase } from '../../../../common/types';
+import { StatusFilterBase } from '../../../../common/types';
+import type { DashboardMigrationFilters } from '../../../../../../common/siem_migrations/dashboards/types';
+
+const STATUS_FILTERS: Record<StatusFilterBase, DashboardMigrationFilters> = {
```

Comment:
Can't this one go in common as well?

Reply:
Yes it can, just don't want to add too much changes in one PR 😄 I will think of how to move this part into a common in one of the following PRs.